src/test/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImplFulfilmentTest.java

```diff
@@ -619,4 +670,41 @@ private CollectionCase selectCollectionCaseForTest(
     }
     return events;
   }
+
+  private void mockCircuitBreakerRun() {
+    doAnswer(
+            new Answer<Object>() {
+              @SuppressWarnings("unchecked")
+              @Override
+              public Object answer(InvocationOnMock invocation) throws Throwable {
+                Object[] args = invocation.getArguments();
+                Supplier<Object> runner = (Supplier<Object>) args[0];
+                return runner.get();
+              }
+            })
+        .when(circuitBreaker)
+        .run(any(), any());
+  }
+
+  private void mockCircuitBreakerFail() {
```

Comment:
As discussed the mockCircuitBreakerRun and Fail methods could be amalgamated into a single method which is actually a minimal implementation of a circuit breaker. If so then it's more than a mock and naming as one would confuse the reader.

Reply:
done